examples/deviceshifu/demo_device/edgedevice-lwm2m/mockdevice/lwm2m-mock.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: leshan-client
+  namespace: deviceshifu
+  labels:
+    app: leshan-client
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: leshan-client
+  template:
+    metadata:
+      labels:
+        app: leshan-client
+    spec:
+      containers:
+      - name: leshan-client
+        image: edgenesis/lwm2m-demo-leshan-client:nightly
```

Comment:
we need to either include this image in the dockerfile directory, add it into makefile or use the official leshan-client